app/controllers/events_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
 
   def random_attendee
     attendes = meetup.attendence(urlname, event_id)
-    person = attendes.shuffle.pop[:member]
+    person = attendes.sample.member
```

Comment:
@vitortalaia do you think we still need the ```attendance``` method on the service? we could just call a ```random attendee``` method from the controller